samples/cellular/modem_shell/overlay-cloud_coap.conf

```diff
@@ -20,3 +20,5 @@ CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE=40
 CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS=y
 CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_NETWORK=y
 CONFIG_NRF_CLOUD_SEND_DEVICE_STATUS_SIM=y
+CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_UI=y
```

Comment:
Currently, mosh only supports sending location data to the cloud. `CONFIG_NRF_CLOUD_ENABLE_SVC_INF_UI_RSRP` seems to be enabled by default, maybe that should be explicitly disabled in both CoAP and MQTT overlays.

Previously, the map card has been enabled only when location support is enabled in mosh, but I don't think that's too important.

Reply:
sure, i will disable it